vita3k/regmgr/src/regmgr.cpp

```diff
@@ -58,15 +59,32 @@ static std::string decryptRegistryFile(const fs::path reg_path) {
     return res;
 }
 
+enum RegType {
+    REG_TYPE_INT,
+    REG_TYPE_STR,
+    REG_TYPE_BIN,
+};
+
 struct RegValue {
     std::string name;
+    RegType type;
     uint32_t size;
-    std::string value;
+    std::vector<char> init_value;
 };
 
 static std::vector<std::string> reg_category_template;
 static std::map<std::string, std::vector<RegValue>> reg_template;
 
+static std::vector<char> convert_int_to_hex_string(const int32_t &value) {
+    // Aloocate size for 4 bytes
+    std::vector<char> res(4);
+
+    // Convert the int to a hex string (4 bytes)
```

Comment:
That's definitely not a hex string

Reply:
is hex string
![image](https://github.com/Vita3K/Vita3K/assets/5261759/5b350b15-9625-4d83-b5b6-27aebc6304b7)
Is hex value writed in char

Reply:
if you log this result to you get this .5p., is hexa value converted to char

Reply:
Code litteraly convert int number to

Reply:
i have removed comment but like i said, i have just follow what princess have learn me for convert binary hexa char to hexa value
is same
for screen category verison is 
decimale = 3502084
hexa = 00 35 70 04 you can see it on screen
char writed .p5.

Reply:
That has absolutely nothing to do with hexadecimal and that's not even a string. It's just that the data is stored as big endian

Reply:
ok, if you said👌